packages/eslint-plugin-i18n/src/rules/interpolation-data.js

```diff
@@ -1,6 +1,45 @@
+const _ = require('lodash');
 const minimatch = require('minimatch');
 const { getKeyValue, get, getLangConfig, areWeUsingUseTranslate } = require('../utils/utils');
 
+const check = (key, countNode, dataNode, config, context, node) => {
+  getLangConfig(config, 'principalLangs').forEach(({ translation }) => {
+    if (!translation) {
+      return;
+    }
+
+    const isPluralized = !!countNode && Array.isArray(config.pluralizedKeys);
+    const translateValue = get(translation, key);
+    const [{ interpolationPattern }] = context.options;
```

Comment:
I'm not sure if this is safe ? 🤔 
Are you sure that options is an array and are you sure the first element in that array is an object ? 
If you are not this will be an execution error.

Reply:
I don't think there is a pb => https://eslint.org/docs/developer-guide/working-with-rules#contextoptions
Yes, options is always an array, and yes interpolationPattern will always be at first because it is the way this rul has to be configured. (I just move some existing code here)